core/src/main/java/com/linecorp/armeria/client/endpoint/FileWatcherRegistry.java

```diff
@@ -74,10 +76,10 @@
         void register(FileWatchRegisterKey watchRegisterKey, Runnable callback) {
             final Path dirPath;
             try {
-                dirPath = watchRegisterKey.getFilePath().toRealPath().getParent();
+                dirPath = watchRegisterKey.filePath().toRealPath().getParent();
             } catch (IOException e) {
                 throw new IllegalArgumentException("failed to locate file " +
```

Comment:
For example, this doesn't have to be capitalized because it's not a complete sentence.